test/unit/services/finance/stripe_charge_service_test.rb

```diff
@@ -7,8 +7,9 @@ class Finance::StripeChargeServiceTest < ActiveSupport::TestCase
     provider_account = FactoryBot.create(:simple_provider, payment_gateway_type: :stripe, payment_gateway_options: { login: 'sk_test_4eC39HqLyjWDarjtT1zdp7dc' })
     buyer_account = FactoryBot.create(:simple_buyer, provider_account: provider_account)
     @gateway = provider_account.payment_gateway(sca: true)
-    @invoice = FactoryBot.create(:invoice, buyer_account: buyer_account, provider_account: provider_account)
     @amount = ThreeScale::Money.new(150.0, 'EUR')
+    line_item = FactoryBot.create(:line_item, cost: amount.amount)
+    @invoice = FactoryBot.create(:invoice, buyer_account: buyer_account, provider_account: provider_account, line_items: [line_item])
```

Comment:
Since the `line_item` is only used once as a value for the `line_items` attribute (which explains what's going on there already :v:), I'd remove the variable assignment and do something like 

```suggestion
    @invoice = FactoryBot.create(
      :invoice,
      buyer_account: buyer_account,
      provider_account: provider_account,
      line_items: [
        FactoryBot.create(:line_item, cost: amount.amount)
      ]
    )
```

Reply:
Just for the sake of maybe future discussions about this topic (and so it doesn't sound like my personal preference 😬), my comment came from past offenses on [RuboCop::Cop::Metrics::AbcSize](https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Metrics/AbcSize). It checks the total number of 
- assignments (anything with `=`)
- branches (anything that jumps out of the current method)
- conditionals (anything that tests logic `if`, `case`, `?`)

The idea is to remove extra code that could create distractions from the main purpose of a method, thus having it cleaner.
For this point, the benefit is that we don't have to pay extra attention to `line_item`, as it's just another attribute of what's really important for the test: the `@invoice` instance variable.